traffic_ops/traffic_ops_golang/login/login.go

```diff
@@ -154,6 +154,42 @@ func LoginHandler(db *sqlx.DB, cfg config.Config) http.HandlerFunc {
 				httpCookie := tocookie.GetCookie(form.Username, defaultCookieDuration, cfg.Secrets[0])
 				http.SetCookie(w, httpCookie)
 
+				var jwtToken *jwt.Token
+				useJwt := r.URL.Query().Get("token-type") == "jwt"
+				var jwtSigned string
+				if useJwt {
```

Comment:
why not just always also set this cookie?

Reply:
good call! might as well